packages/riverpod/lib/src/common.dart

```diff
@@ -157,12 +157,24 @@ abstract class AsyncValue<T> {
   ///     });
   ///   }
   /// }
+  ///
+  /// when predicate is provided, and it is false; rethrown.
```

Comment:
You've edited the code-snippet. Explanation should be outside of the ` ``` `

Also, could you expand on it a bit more?

Reply:
Thanks, I added it.